src/com/pinktwins/elephant/ElephantWindow.java

```diff
@@ -846,6 +873,7 @@ private void createMenu() {
 		file.add(menuItem("New Notebook", KeyEvent.VK_N, menuMask | KeyEvent.SHIFT_DOWN_MASK, newNotebookAction));
 		file.add(menuItem("New Tag", KeyEvent.VK_T, menuMask | KeyEvent.CTRL_DOWN_MASK, newTagAction));
 		file.add(menuItem("New Elephant Window", KeyEvent.VK_N, menuMask | KeyEvent.ALT_DOWN_MASK, newWindowAction));
+		file.add(menuItem("Switch Note Location", KeyEvent.VK_L, menuMask, switchNoteLocationAction));
```

Comment:
.. on mac, but on Windows menuMask equals CTRL already.. so it needs both CTRL_DOWN_MASK | ALT_DOWN_MASK propably.